srv/ExposureAuto.srv

```diff
@@ -0,0 +1,3 @@
+bool enable
```

Comment:
It can be `std_srvs/SetBool`.
http://docs.ros.org/melodic/api/std_srvs/html/srv/SetBool.html

Reply:
Thank you. I will use it.